guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p><strong>Note:</strong> In the absence of <code>arcpy</code>, the <a href="https://pypi.org/project/Fiona/" target="_blank">Fiona</a> package must be present in your current conda environment to perform this operation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>